cpp/generator.cpp

```diff
@@ -102,15 +101,16 @@ std::vector<int64_t> tokenize(const char *input) {
 }
 
 /* Simulated Byt5 detokenizer that reverses the effects of its tokenizer */
-std::string detokenize(const std::vector<int64_t> &tokens) {
```

Comment:
Detokenization seems to have nothing to do with the scores? Why do we pass the scores into this function?

Reply:
Right, detokenization does not need the scores as input. Technically we can factor that out and only pass the tokens into it, let me change this.